content/en/continuous_integration/intelligent_test_runner/how_it_works.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How It Works
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Datadog Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+
+## What is Intelligent Test Runner?
+
+Intelligent Test Runner is Datadog's test impact analysis solution. Test impact analysis is a technique that has been getting popular in the past couple of decades, but is usually hard and time-consuming to implement. Intelligent Test Runner cuts through that complexity with a simple setup process.
+
+Test impact analysis maps each test to the set of code files in your repository that your test exercises, with the goal of reducing the time taken by the testing phase of your CI. This is achieved by analyzing the code changes you are trying to test, and skipping the tests that do not impact your code changes.
+
+An extreme example of this would be a Pull Request that is only changing a typo on a README file. For that PR, running all tests doesn't provide much value. On the contrary, if you have any flaky tests your CI might fail and you might have to retry several times before being able to merge. This is a waste of both developer and CI time. With Intelligent Test Runner, a PR that is only changing a README file would skip all tests.
+
+## What sets it apart
+
+Some test selection solutions try to make up for the lack of code coverage information by trying to infer in a probabilistic fashion which tests are relevant through the use of Machine Learning. This results in a test selection mechanism that can skip tests that were relevant, and cause build failures in your default branch.
```

Comment:
is it the first time we mention "code coverage" in this section? It seems we were referring to it as "test impact analysis". It might surprising for users to read code coverage now 

Maybe this?
```suggestion
Some test selection solutions make up for the lack of per test impact analysis through the use of Machine Learning techniques, which infer what tests are relevant for a given code change in a probabilistic fashion. This results in a test selection mechanism that can skip tests that were relevant, causing build failures in your default branch.
```

Reply:
@juan-fernandez I like the change. Think we should also add this other aspect at the end? 

"ML-based techniques also typically require longer periods of data collection before they're able to work. Intelligent Test Runner begins working immediately after a baseline of code coverage is gathered."